wg-data/charter.md

```diff
@@ -0,0 +1,70 @@
+# WG Data Charter
+
+This charter adheres to the conventions, roles, and organisation management outlined in [wg-governance] for the Working Group "Data".
+
+## Scope
+
+The WG "Data" is focused on enhancing the support for data/metadata-related tasks within Kubeflow, with a specific focus on the [Spark operator](https://github.com/kubeflow/community/pull/672) and [Model Registry](https://github.com/kubeflow/kubeflow/issues/7396).
+The group aims to simplify and improve data processing between various stages of ML lifecycle. For example, from Data Preparation to model training and fine-tuning.
+The group also aims to facilitate the ML model's metadata management, while ensuring seamless integration with other Kubeflow components.
+
+An additional goal of the group is to offer a common ground for data/metadata-related topics in the MLOps orbit that didn't have a more specific working group yet, so they can "incubate as one", coherent effort.
+
+For example: Data Preparation, Feature Store, and Model Registry have been recently discussed in the Kubeflow community while not mature enough yet to have their own working group, they can be nurtured together as part of this WG.
+
+### In scope
+
+#### Code, Binaries, and Other relevant assets
+
```

Comment:
```suggestion
- Onboarding and maintenance of Feast for feature historical feature extraction and online feature serving.
```

Reply:
if you don't mind i didn't put at line 18 as the first bulletpoint :)

done with fa3c3187d1d731bfe723c9c9b633a868baad051c